nsdb-core/src/main/scala/io/radicalbit/nsdb/actors/MetricReaderActor.scala

```diff
@@ -130,12 +129,13 @@ class MetricReaderActor(val basePath: String, nodeName: String, val db: String,
     * @param aggregationFunction the aggregate function corresponding to the aggregation operator (sum, count ecc.) contained in the query.
```

Comment:
should we add **statement** as param to scaladocs as well?

Reply:
Yes will fix it